tests/README.md

```diff
@@ -29,6 +29,14 @@ environment variable. For example:
 ANSIBLE_REMOTE_USER=root IPA_SERVER_HOST=<ipaserver_host_or_ip> pytest
 ```
 
+If you want to use ssh with password, you must set `IPA_SSH_PASSWORD`
+environment variable. For example:
+
+```
+IPA_SSH_PASSWORD=SomeSecret IPA_SERVER_HOST=<ipaserver_host_or_ip> pytest
```

Comment:
Just to keep consistence I'd use `IPA_SSH_PASSWORD=<ipaserver_ssh_password>`

Reply:
Changed.